homeassistant/components/apsystems_local/manifest.json

```
@@ -0,0 +1,13 @@
+{
+  "domain": "apsystems_local",
+  "name": "APsystems Local API",
```

It's just APSystems. It's an implementation detail that it's local and should not be reflected in name or domain.
```suggestion
  "name": "APsystems",
```